### ISSUE TITLE
Single-element menu disappears when I click on it

### ISSUE THREAD

**Issue by xipmix** [volumio/Volumio2-UI#706]
In Cog->PlaybackOptions, there is the 'Mixer Control Name' setting.
In my Pi I have just the onboard audio

```
$ cat /proc/asound/cards
 0 [ALSA           ]: bcm2835_alsa - bcm2835 ALSA
                      bcm2835 ALSA
$ cat /proc/asound/devices 
  0: [ 0]   : control
  1:        : sequencer
 16: [ 0- 0]: digital audio playback
 17: [ 0- 1]: digital audio playback
 18: [ 0- 2]: digital audio playback
 33:        : timer
$ amixer -c 0 scontents
Simple mixer control 'PCM',0
  Capabilities: pvolume pvolume-joined pswitch pswitch-joined
  Playback channels: Mono
  Limits: Playback -10239 - 400
  Mono: Playback -2336 [74%] [-23.36dB] [on]

```
so there is only one menu item, 'PCM'.

When I click on the menu widget to see what the other options might be,
the menu disappears (see screenshot). Clicking in the spot where it used to be brings it back.

Similar to #692 ?

![gone](https://user-images.githubusercontent.com/2238100/71760953-1880bf00-2f19-11ea-9aa1-43d372efefa6.jpg)

**Comment by xipmix** [comment]
Strange, this seems to have gone away too.

Browser is FF71.0 on ubuntu16.04. Will test with others, if it comes back

**Comment by xipmix** [comment]
Looks like it is fixed by volumio/Volumio2/pull/1872
